app/javascript/css/main.scss

```diff
@@ -515,3 +515,54 @@ a.disabled {
   color: var(--primary-color);
   border-bottom: 1px dashed var(--primary-color);
 }
+
+.delete-icon {
+  color: var(--status-red);
+}
+
+.inbox-sender {
+  font-size: 16px;
+  font-weight: 600;
+}
+
+.tabs {
+  margin-top: 3ch;
+}
+
+.tab {
+  border: none;
+  margin-right: 0.25ch;
+  padding: 1.25ch 1ch 1ch;
+  font-size: 1rem;
+  font-weight: 600;
+  color: var(--grey-140);
+  background: var(--grey-020);
+  border-radius: 5px 5px 0 0;
+  border-bottom: 1px solid #dee2e6;
+}
+
+.tab.active {
+  background: #ffffff;;
+  color: var(--primary-color);
+  border-top: 1px solid #dee2e6;
+  border-left: 1px solid #dee2e6;
+  border-right: 1px solid #dee2e6;
+  border-bottom: none;
+}
+
+.no-underline {
+  text-decoration: none;
+}
+
+.no-underline:hover {
+  text-decoration: none;
+}
+
+.msg-inbox:hover {
```

Comment:
updated tab css is below, you'll also have to add a new color of the teritary-tint
`.tab-holder {
  margin-bottom: 6px;
}

.tab {
  border: none;
  margin-right: 0.25ch;
  padding: 1.25ch 1ch 1ch;
  font-size: 1rem;
  font-weight: 600;
  color: var(--default-font-color);
  background: var(--tertiary-tint);
  border-radius: 5px 5px 0 0;
  border-bottom: 1px solid var(--grey-060);
}

.tab.active {
  background: #ffffff;;
  color: var(--primary-color);
  border-top: 1px solid var(--grey-060);
  border-left: 1px solid var(--grey-060);
  border-right: 1px solid var(--grey-060);
  border-bottom: 1px solid var(--grey-000);
}

.tab-container {
  margin-top: 5px;
  border: 1px solid var(--grey-060);
}`